### ISSUE TITLE
Responsive Behavior

### ISSUE THREAD

**Issue by jbkunst** [cuche27/combineWidgets#1]
Hi @cuche27 !

Great package!

I think it's better place to keep the discussion here.

About the question in ramnathv/htmlwidgets/pull/245.  Yes, I'm referencing to change automatically the layout if the screen size change.

**Comment by timelyportfolio** [comment]
@jbkunst, agreed, but assuming @cuche27 confirms, I think this behavior is outside the scope of the `flex` solution provided by `combineWidgets`.  In that case, I think you are much better off using a grid-based CSS framework, such as Bootstrap, but I could easily be completely wrong :)

https://github.com/ramnathv/htmlwidgets/pull/245#issuecomment-262308542

**Comment by FrancoisGuillem** [comment]
Indeed, my first intention was to fill all the space available and to have an easy way to specify how to share it between widgets. The idea was to have something similar to `par(mfrow=c(x,y))` for htmlwidgets.

Anyway, the idea is interesting and I will think about it !
